src/plone/volto/distributions/default/content/content/plone_site_root/data.json

```diff
@@ -0,0 +1,37 @@
+{
+  "@id": "/default",
+  "@type": "Plone Site",
+  "UID": "plone_site_root",
+  "allow_discussion": null,
+  "blocks": {},
```

Comment:
Shouldn't there be some default blocks?

Reply:
Yes, we need to have the default content here.